### PR TITLE
Add STABILIZED_[YAW, ROLL, PITCH] for logic_condition

### PR DESCRIPTION
**Pull request by joleeee**
Let me know what needs to be changed.
Original issue: https://github.com/iNavFlight/inav/issues/5617

### CODE REVIEW

**Comment by joleeee** [comment]
@DzikuVx Perhaps you could give some input on this? I understand you are the author of the logic conditions and probably have some opinions on implementation. I have tested a version of this where I use 4 modded servos as the motors on a rover, and it works well. 

Also a picture of my servo setup:
unknown

**Comment by DzikuVx** [comment]
Quite impressive!

File: src/main/common/logic_condition.h [hunk 1]
```diff
@@ -90,6 +90,9 @@ typedef enum {
     LOGIC_CONDITION_OPERAND_FLIGHT_IS_WP, // 0/1                            // 23
     LOGIC_CONDITION_OPERAND_FLIGHT_IS_LANDING, // 0/1                       // 24
     LOGIC_CONDITION_OPERAND_FLIGHT_IS_FAILSAFE, // 0/1                      // 25
+    LOGIC_CONDITION_OPERAND_FLIGHT_STABILIZED_YAW,                          // 26
```
Comment (DzikuVx) [inline src/main/common/logic_condition.h]: @joleeee please reorder to ROLL PITCH YAW and we use this order everywhere.

**Review by DzikuVx — CHANGES_REQUESTED** [review]

**Comment by DzikuVx** [comment]
also @joleeee do you think you could make the Configurator part? It's should we as simnple as adding those entries in one file

**Comment by joleeee** [comment]
Done. https://github.com/iNavFlight/inav-configurator/pull/971

**Review by DzikuVx — APPROVED** [review]

**Comment by DzikuVx** [comment]
thanks a lot

**Pull request merged**